modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -168,80 +204,184 @@ function indexOf($comp, $array)
       }
     }
 
-    $episode_index = indexOf(function ($value) use ($episode_id) {
-      return $value['episode_id'] == $episode_id;
+    $episodeIndex = indexOf(function ($value) use ($episodeId) {
+      return $value['episode_id'] == $episodeId;
     }, $series);
 
-    if (is_null($episode_index)) {
+    if (is_null($episodeIndex)) {
       return array();
     }
 
-    $episode_offset = $episode_index + 1;
+    $episodeOffset = $episodeIndex + 1;
 
-    $episodes = array_slice($series, $episode_offset, $number);
+    $episodes = array_slice($series, $episodeOffset, $numberOfNextEpisodes);
 
     return $episodes;
   }
+
   /**
    * TranslateNode function
    *
    * @param NodeInterface $node
    *
-   * @return $node
-   */
-  private function translateNode(NodeInterface $node)
-  {
-    return $node->hasTranslation($this->lang) ? $node->getTranslation($this->lang) : $node;
+   * @return NodeInterface
+  */
+  private function translateNode($node) {
+    return $node->hasTranslation($this->language) ? $node->getTranslation($this->language) : $node;
   }
 
   /**
-   * Get nids
+   * Loads term for ID
+   *
+   * @param int $prisonId
    *
-   * @return void
-   */
-  private function getSeriesContentNodeIds($series_id, $number, $offset, $prison)
-  {
-    $results = $this->entity_query->get('node')
+   * @return EntityInterface
+  */
+  private function getTermForRequest($termId) {
+    $term = $this->termStorage->load($termId);
+
+    if (!$term) {
+      throw new NotFoundHttpException(
+        'Term not found',
+        null,
+        404
+      );
+    }
+
+    return $term;
+  }
+
+  /**
+   * Get Prison Categories for a Drupal term object
+   *
+   * @param EntityInterface $term
+   * @return int[]
+  */
+  private function getPrisonCategoriesForTerm($term) {
```

Comment:
Maybe could change the name and remove references to Term? That way it can easily be a util function for content as well?

Reply:
Updated 👍